### PR TITLE
Web style

### PR DESCRIPTION
**Pull request by Keelando**
First major change to website gui style. Nothing is permanent, and I am highly open to feedback.

### CODE REVIEW

File: web/lattice_main/views.py [hunk 1]
```diff
@@ -17,11 +20,14 @@ def upload_circuit(request):
...
-    print("post-request",request.FILES['circuit'].name)
 
+    #print("post-request",request.FILES['circuit'].name)
     file_ext = request.FILES['circuit'].name.split('.')[-1]
-    circuit_tmp_save_location = os.path.join('/tmp','%s.%s'%(uuid.uuid4(),file_ext))
+    # Save the file to a tmp dir so that pyzx can read it
+    ### for Linux deployment ###
+    #circuit_tmp_save_location = os.path.join('/tmp','%s.%s'%(uuid.uuid4(),file_ext))
+    ### Windows Testing ###
+    circuit_tmp_save_location = os.path.join('C:/Users/Keelan/Documents/Websites and Programming Projects/Lattice Surgery/temp server files','%s.%s'%(uuid.uuid4(),file_ext))
```
Comment (gwwatkin) [inline web/lattice_main/views.py]: We can probably find a way to not have to hard code paths :thinking:

File: web/lattice_main/templates/lattice_main/base.html [hunk 2]
```diff
@@ -1,15 +1,112 @@
...
+
+  <footer class="foot-bg">
+  <!-- commenting these out untill thery are implemented  -->
+    <div class="container">
+      <div class="flex-container justify-content-center">
+        <div>
+           <a href="https://github.com/latticesurgery-com/lattice-surgery-compiler/"><i class="fab fa-github fa-3x"></i></a>
+        </div>
+        <div>
+          <a href="https://google.ca"><i class="fas fa-book-open fa-3x"></i></a>
```
Comment (gwwatkin) [inline web/lattice_main/templates/lattice_main/base.html:75]: I suppose we need to put a `mailto: ` or make a form

**Review by gwwatkin — APPROVED** [review]
Looks good, can merge. Or you can slip in the checkbox for the Litinski Transform as discussed

File: web/lattice_main/views.py [hunk 3]
```diff
@@ -32,8 +32,13 @@ def view_compiled(request):
     with open(circuit_tmp_save_location, 'wb') as output_file:
         shutil.copyfileobj(input_circuit_file, output_file)
 
+    apply_litinski_transform = True
+    if "litinsky" not in request.POST:
```
Comment (gwwatkin) [inline web/lattice_main/views.py:36]:
```suggestion
    if "litinski" not in request.POST:
```

File: web/lattice_main/templates/lattice_main/upload_circuit.html [hunk 4]
```diff
@@ -43,7 +48,11 @@ <h2 id="get-started" class='anchor'>Get Started - Upload a circuit</h2>
                   <input type="submit" class="btn btn-info" value="Go!" />
                 </div>
                 <input type="hidden" name="localcircuit" value="yes">
-                
+              </div>
+
+              <div class="form-check">
+                <input type="checkbox" class="form-check-input" id="litinsky_check1" name='litinsky' value='true' checked>
+                <label class="form-check-label" for="litinsky_check1">Litinsky Transform</label>
```
Comment (gwwatkin) [inline web/lattice_main/templates/lattice_main/upload_circuit.html:55]:
```suggestion
                <label class="form-check-label" for="litinski_check1">Litinski Transform</label>
```

File: web/lattice_main/templates/lattice_main/upload_circuit.html [hunk 5]
```diff
@@ -43,7 +48,11 @@ <h2 id="get-started" class='anchor'>Get Started - Upload a circuit</h2>
                   <input type="submit" class="btn btn-info" value="Go!" />
                 </div>
                 <input type="hidden" name="localcircuit" value="yes">
-                
+              </div>
+
+              <div class="form-check">
+                <input type="checkbox" class="form-check-input" id="litinsky_check1" name='litinsky' value='true' checked>
```
Comment (gwwatkin) [inline web/lattice_main/templates/lattice_main/upload_circuit.html:54]:
```suggestion
              <input type="checkbox" class="form-check-input" id="litinski_check1" name='litinski' value='true' checked>
```

File: web/lattice_main/templates/lattice_main/upload_circuit.html [hunk 6]
```diff
@@ -18,11 +18,16 @@ <h2 id="get-started" class='anchor'>Get Started - Upload a circuit</h2>
...
+
+
                 <div class="input-group-append">
                   <input type="submit" class="btn btn-primary" value="Go!" />
                 </div>
               </div>
-              
+                <div class="form-check">
+                  <input type="checkbox" class="form-check-input" id="litinsky_check1" name='litinsky' value='true' checked>
+                  <label class="form-check-label" for="litinsky_check1">Litinsky Transform</label>
```
Comment (gwwatkin) [inline web/lattice_main/templates/lattice_main/upload_circuit.html:29]:
```suggestion
                   <label class="form-check-label" for="litinski_check1">Litinski Transform</label>
```

File: web/lattice_main/templates/lattice_main/upload_circuit.html [hunk 7]
```diff
@@ -18,11 +18,16 @@ <h2 id="get-started" class='anchor'>Get Started - Upload a circuit</h2>
...
                 <input id="circuit" class="form-control circuit" name="circuit" type="file"/>
+
+
                 <div class="input-group-append">
                   <input type="submit" class="btn btn-primary" value="Go!" />
                 </div>
               </div>
-              
+                <div class="form-check">
+                  <input type="checkbox" class="form-check-input" id="litinsky_check1" name='litinsky' value='true' checked>
```
Comment (gwwatkin) [inline web/lattice_main/templates/lattice_main/upload_circuit.html:28]:
```suggestion
                  <input type="checkbox" class="form-check-input" id="litinski_check1" name='litinski' value='true' checked>
```

File: web/lattice_main/views.py (hunk 1 above)
Comment (Keelando) [reply web/lattice_main/views.py]: Yes, the above is outdated now. I created a folder internal to the project in /web/lattice_main/tmp which is used instead. I think I'm still using the relative to absolute path tool os.path.abspath as you used previously.

**Pull request merged**